samples/opencv_cuda_bg_remover_mog2/bgremover.py

```diff
@@ -35,8 +35,9 @@ def process_frame(self, buffer: Gst.Buffer, frame_meta: NvDsFrameMeta):
         :param buffer: Gstreamer buffer with this frame's data.
         :param frame_meta: This frame's metadata.
         """
+        stream = self.get_cuda_stream()
```

Comment:
Replace `stream=artist.stream` with `stream=stream` in the function gaussian_filter.apply, back_sub.apply, ref_frame.copyTo.
It looks more logical to me.

Reply:
Fixed.